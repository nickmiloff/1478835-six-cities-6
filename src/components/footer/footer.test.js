import React from 'react';
import {render} from '@testing-library/react';
import {TestHistory as Test} from '../test/test';

import Footer from './footer';

describe(`Footer component test`, () => {
  it(`'Footer component' should render correctly`, () => {
    const {container} = render(
        <Test>
          <Footer />
        </Test>
    );

    expect(container).toMatchSnapshot();
  });
});
