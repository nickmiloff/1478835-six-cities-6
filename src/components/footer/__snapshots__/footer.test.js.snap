// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component test 'Footer component' should render correctly 1`] = `
<div>
  <footer
    class="footer container"
  >
    <a
      class="footer__logo-link"
      href="/"
    >
      <img
        alt="6 cities logo"
        class="footer__logo"
        height="33"
        src="img/logo.svg"
        width="64"
      />
    </a>
  </footer>
</div>
`;
